docs/core/install/linux.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Install .NET Core and Linux Distributions
+description: Learn about what Linux distributions support installing .NET Core on Linux.
+author: thraka
+ms.author: adegeo
+ms.date: 06/01/2020
+---
+
+# Install .NET Core on Linux
+
+.NET Core is available on different Linux distributions. Most Linux platforms and distributions have a major release each year, and most provide a package manager that is used to install .NET Core. This article describes what is currently supported and which package manager is used.
+
+The rest of this article is a breakdown of each major Linux distribution that .NET Core supports. All .NET Core releases remain supported until either the version of [.NET Core reaches end-of-support](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) or the Linux distribution reaches end-of-life.
+
+For the best compatibility, choose a long-term release (LTS) version.
+
+## Unsupported releases
+
+The following versions of .NET Core are ❌ no longer supported. The downloads for these still remain published:
+
+- 3.0
+- 2.2
+- 2.0
+
+These unsupported versions aren't detailed in the sections below and your mileage may vary if you try to install them.
+
+## CentOS
+
+CentOS 7 uses Yum as a package manager and CentOS 8 uses DNF.
+
+The following table is a list of currently supported .NET Core releases on both CentOS 7 and CentOS 8. These versions remain supported until either the version of [.NET Core reaches end-of-support](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) or the version of CentOS is no longer supported.
+
+| CentOS                   | .NET Core 2.1 | .NET Core 3.1 | .NET 5 Preview |
+|--------------------------|---------------|---------------|----------------|
+| ✔️ [8](linux-centos.md#centos-8-) | ✔️ 2.1        | ✔️ 3.1        | ✔️ 5.0 Preview |
```

Comment:
I was using 5.0 as a stand-in to look at how these formats scale for the future, you can't actually get them from the package repos--you can install it manually (so this page I guess is still accurate?) but I think I posted some misleading suggestions, sorry. I think these tables should probably only show 2.1 and 3.1 for simplicity?